SoftLayer/managers/billing.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
As the mostly managed Manager modules their own billing stuff, maybe it's better to move this logic into the Network Manager, but in any case, keep in mind to add tests for that.